frontend/public/components/routes/create-route.tsx

```diff
@@ -541,57 +555,64 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
-                          TLS certificates for edge and re-encrypt termination. If not specified,
-                          the router&apos;s default certificate is used.
+                          {t(
+                            'create-route~TLS certificates for edge and re-encrypt termination. If not specified, the router&apos;s default certificate is used.',
+                          )}
                         </p>
                       </div>
                       <div className="form-group co-create-route__certificate">
                         <DroppableFileInput
                           onChange={this.onCertificateChange}
                           inputFileData={this.state.certificate}
                           id="certificate"
-                          label="Certificate"
-                          inputFieldHelpText="The PEM format certificate. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard."
+                          label={t('create-route~Certificate')}
+                          inputFieldHelpText={t(
+                            'create-route~The PEM format certificate. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard.',
+                          )}
                         />
                       </div>
                       <div className="form-group co-create-route__private-key">
                         <DroppableFileInput
                           onChange={this.onPrivateKeyChange}
                           inputFileData={this.state.key}
                           id="private-key"
-                          label="Private Key"
-                          inputFieldHelpText="The PEM format key. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard."
+                          label={t('create-route~Private Key')}
+                          inputFieldHelpText={t(
+                            'create-route~The PEM format key. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard.',
+                          )}
                         />
                       </div>
                       <div className="form-group co-create-route__caCertificate">
                         <DroppableFileInput
                           onChange={this.onCaCertificateChange}
                           inputFileData={this.state.caCertificate}
                           id="ca-certificate"
-                          label="CA Certificate"
-                          inputFieldHelpText="The PEM format CA certificate chain. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard."
+                          label={t('create-route~CA Certificate')}
+                          inputFieldHelpText={t(
+                            'create-route~The PEM format CA certificate chain. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard.',
```

Comment:
Remove HTML entity.

Reply:
The HTML entity is the escaped string of &.
Changed HTML entity to & in file.
Thanks.